metron-platform/metron-common/src/test/java/org/apache/metron/common/stellar/StellarTest.java

```diff
@@ -288,6 +289,26 @@ public void testIfThenElse() {
   }
 
   @Test
+  public void testInNotIN(){
+    HashMap variables = new HashMap<>();
+    boolean thrown = false;
+    try{
+      run("in in ['','in']" ,variables );
```

Comment:
We could probably `Assert.assertFalse((boolean)run("in in ['', 'in']", variables))` and ensure expected behavior.  Similarly for the other case.